docs/support.rst

```diff
@@ -0,0 +1,10 @@
+Support and Communication
+-------------------------
+All bugs, concerns and enhancement requests for this software can be submitted here: https://github.com/poldracklab/fmriprep/issues.
```

Comment:
This looks like an old link ! :laughing:

Reply:
Good catch! Should be fixed now.